nextjs-logs-manager/app/login/actions.ts

```
@@ -0,0 +1,34 @@
+import { createSession } from "../_lib/session";
+import { PrismaClient } from "@prisma/client";
+
+export const Login = async (prevState: any, formData: FormData) => {
+  const prisma = new PrismaClient();
+
+  // Validate that the username and password fields are not empty
+  const form_fields = ["username", "password"];
+  for (const field of form_fields) {
+    if (!formData.get(field)) {
+      {
+        message: `The field ${field} can't be empty`;
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)